packages/components/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,15 @@
+import { Elevation, mapElevationToBorder, spacingsPx } from '@trezor/theme';
+import { useElevation } from '../ElevationContext/ElevationContext';
+import styled from 'styled-components';
+
+const Line = styled.div<{ elevation: Elevation }>`
+    width: 100%;
+    height: 1px;
+    background: ${({ theme, elevation }) => mapElevationToBorder({ theme, elevation })};
+    margin: ${spacingsPx.md} 0;
+`;
+
+export const Divider = () => {
```

Comment:
:clap:

Reply:
optional/nit: horizontal / vertical variability

Reply:
This will be fixed while we introduce layout props for every DS component that can need margin.